sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -0,0 +1,459 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/// <reference lib="esnext.asynciterable" />
+
+import {
+  operationOptionsToRequestOptionsBase,
```

Comment:
maybe we could use the new generator at some point to avoid needing to do this helper in the convenience layer?

Reply:
that would be great! Should I log an issue? How would you approach this?

Reply:
An issue is fine- we usually just regenerate and fix the build errors, I think. Maybe you could move to /generated/ at the same time?

Reply:
Can we do this later? Seems like it's creeping on another scope, if that makes sense.

Reply:
Yes absolutely. As mentioned, an issue is fine. 👍 

"at some point" in the first comment was supposed to imply "not needed for this review"; I'll be more explicit to avoid confusion in the future. 😄